.gitignore

```diff
@@ -1,3 +1,3 @@
-build/
-isar/
+/build
+/isar
```

Comment:
Unrelated change.

Reply:
The isar needs to be change for the new patches directory.  I will correct it.